Tests/js/views/fields/assets/ez-author-editview-tests.js

```diff
@@ -372,100 +392,268 @@ YUI.add('ez-author-editview-tests', function (Y) {
+                            }
+                        }, this);
                     });
                 });
+            });
 
-                this.wait();
-            },
+            this.wait();
+        },
 
-            _fillAuthor: function (author, i) {
-                var container = this.view.get('container');
+        _fillAuthor: function (author, i) {
+            var container = this.view.get('container');
 
-                container.all('.ez-field-author-name').item(i).set('value', author.name);
-                container.all('.ez-field-author-name').item(i).simulate('blur');
-                container.all('.ez-field-author-email').item(i).set('value', author.email);
-                container.all('.ez-field-author-email').item(i).simulate('blur');
-            },
+            container.all('.ez-field-author-name').item(i).set('value', author.name);
+            container.all('.ez-field-author-name').item(i).simulate('blur');
+            container.all('.ez-field-author-email').item(i).set('value', author.email);
+            container.all('.ez-field-author-email').item(i).simulate('blur');
+        },
 
-            _assertCorrectFieldValue: function (fieldValue, msg) {
-                Y.Assert.areEqual(
-                    2, fieldValue.length,
-                    "The field value should contain only the valid author"
-                );
-                Y.Assert.areEqual(this.newValue[0].email, fieldValue[0].email, msg);
-                Y.Assert.areEqual(this.newValue[0].name, fieldValue[0].name, msg);
-
-                Y.Assert.areEqual(this.newValue[2].email, fieldValue[1].email, msg);
-                Y.Assert.areEqual(this.newValue[2].name, fieldValue[1].name, msg);
-            },
-        })
-    );
+        _assertCorrectFieldValue: function (fieldValue, msg) {
+            Y.Assert.areEqual(
+                2, fieldValue.length,
+                "The field value should contain only the valid author"
+            );
+            Y.Assert.areEqual(this.newValue[0].email, fieldValue[0].email, msg);
+            Y.Assert.areEqual(this.newValue[0].name, fieldValue[0].name, msg);
+
+            Y.Assert.areEqual(this.newValue[2].email, fieldValue[1].email, msg);
+            Y.Assert.areEqual(this.newValue[2].name, fieldValue[1].name, msg);
+        },
+
+    });
     Y.Test.Runner.add(getFieldTest);
 
-    getEmptyFieldTest = new Y.Test.Case(
-        Y.merge(Y.eZ.Test.GetFieldTests, {
-            fieldDefinition: {isRequired: false},
-            ViewConstructor: Y.eZ.AuthorEditView,
-            newValue: "",
-            valuesArray: [],
-
-            _assertCorrectFieldValue: function (fieldValue, msg) {
-                Y.Assert.isArray(fieldValue, 'fieldValue should be an array');
-                Y.Assert.areEqual(fieldValue.length, 0,  msg);
-            },
-        })
-    );
+    getEmptyFieldTest = new Y.Test.Case({
+        setUp: function () {
+            if (!this.content) {
```

Comment:
What's the reason of that? I guess it should be always initiated in `setUp` and destroyed in `tearDown` so there shouldn't be situation when `this.content` must be checked if it already exist.